blocks/identity-block/intl.json

```diff
@@ -1273,5 +1273,20 @@
 		"vi": "Tên người dùng",
 		"zh-CN": "用户名",
 		"zh-TW": "使用者名稱"
+	},
+	"identity-block.bot-protection-error": {
```

Comment:
Hi @Ed, As part of the designs we have an additional reCaptcha state when it expired.